test/Microsoft.AspNetCore.AzureAppServicesIntegration.Tests/Microsoft.AspNetCore.AzureAppServicesIntegration.Tests.csproj

```diff
@@ -20,4 +20,8 @@
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I think we've been adding these in because otherwise you're just fighting VS, which will keep trying to add it back. It is apparently harmless.